private/model/api/docstring.go

```diff
@@ -8,6 +8,8 @@ import (
 	"os"
 	"regexp"
 	"strings"
+
+	xhtml "golang.org/x/net/html"
```

Comment:
Need to take a look at how to prevent the `golang.org/x/net/html` being included as a runtime dependency. I have PR #891 to add `codegen` build tags to the `/private/mode/api` packages to prevent them causing their dependencies to leak into the runtime.

Though I think this is a strong argument for splitting the code generator logic out of the SDK. Doing so would prevent polluting the SDK's runtime environment with code gen deps.